packages/docusaurus-theme-classic/src/theme/DocSidebar/index.tsx

```diff
@@ -187,6 +209,7 @@ function DocSidebar({
   path,
   sidebar,
   sidebarCollapsible = true,
+  autoCollapseSidebar = true,
```

Comment:
this is a behavior change, I'd rather make it opt-in.

Also we tend to avoid using default values in JSX now (as it spread default values in multiple places), and rely on theme config validation/normalization system. If the theme has a new config, it must be added to the Joi validation schema with appropriate default value and tests.

Reply:
I know made it opt-in you now have to set `autoCollapseSidebar` to true in the themeConfig